xbmc/application/Application.cpp

```diff
@@ -1018,9 +1018,7 @@ bool CApplication::OnAction(const CAction &action)
   if (action.GetID() == ACTION_HDR_TOGGLE)
   {
     // Only enables manual HDR toggle if no video is playing or auto HDR switch is disabled
-    if (appPlayer->IsPlayingVideo() &&
-        CServiceBroker::GetSettingsComponent()->GetSettings()->GetBool(
-            CServiceBroker::GetWinSystem()->SETTING_WINSYSTEM_IS_HDR_DISPLAY))
+    if (appPlayer->IsPlayingVideo() && CServiceBroker::GetWinSystem()->IsHDRDisplaySettingEnabled())
```

Comment:
That condition seems odd, does it make sense to allow manual HDR switch during playback. But not relevant for the PR, you kept the logic as is.

Reply:
Not relevant for this PR  as not changes functionality but anyway is correct:

If "Use HDR display capabilities" setting is enabled (which it involves auto HDR switch enabled) and you turn manually HDR ON in a SDR content, auto switch code turns it intermediately again OFF. 

Note that if content is HDR, then HDR is already ON (because auto switch takes care of it).

Conclusion: manual HDR toggle is disabled when playing video and automatic HDR toggle is enabled, but still possible toggle HDR on/off in GUI.